templates/legal/ubuntu-advantage-service-description/ros-esm-service-description/_base_legal_markdown.html

```diff
@@ -0,0 +1,26 @@
+{% extends "templates/base.html" %}
+{% block title %}Ubuntu Advantageサービス範囲{% endblock title %}
+{% block meta_description %}Service description of Ubuntu Advantage - Japanese{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1sEcjyv8Vk4D77EeuyrwtIfllZs5S7uG-YbrEbUTfaqE/edit#{% endblock meta_copydoc %}
```

Comment:
I'm not sure we need any of this - it's not relevant to this page, and they're all defined in `ros-esm-service-description/index.html`